README.md

```diff
@@ -1,35 +1,68 @@
-# kong-docker
+# kong
 
-The Docker image generated by the Dockerfile on this repository can easily install Kong plugins from the source code.
+## Kong with custom plugin
 
-A Kong configuration file containing the plugin names is automatically generated when the Docker container is started. The plugin description file (.rockspec) must be valid following the specifications in https://github.com/luarocks/luarocks/wiki/Rockspec-format.
+The Docker image generated by the Dockerfile contains a kong custom plugin that extract roles from a JWT token and make a request for a XACML Policy Decision Point (PDP)
 
+An example of how to associate a previously created service to the plugin:
 
-Plugins can be loaded from volumes in a docker-compose description file. The configuration below shows how the image can be used.
+``` sh
+kong="http://kong:8001"
+service_name="service_example"
+url_pdp="http://auth:5000/pdp"
+
+curl -X POST \
+--url ${kong}/services/${service_name}/plugins/ \
+--data "name=pepkong" \
+--data "config.pdpUrl=${url_pdp}"
+```
+
+The configuration below shows how the image can be used with docker-compose:
 
 ``` yml
-version: "3"
+version: '2.1'
 services:
-  kong-database:
-    image: postgres:9.4
-    container_name: kong-database
-    ports:
-      - "5433:5432"
+
+  db:
+    image: postgres:9.5.21-alpine
```

Comment:
Maybe to use the same image used in dojot/docker-compose?

Reply:
I upgraded it in https://github.com/dojot/docker-compose/pull/149,

Reply:
Ok, maybe to get from dojot repository in Docker Hub?

Reply:
Ok :)